src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post">
+      <fieldset class="blocks">
+        <legend>Personal information</legend>
+        <label class="inblock-element">
+          Surname:
+          <input 
+          type="text"
+          autocomplete="off"
+          required
+          name="surname">
+        </label>
+        <label class="inblock-element">
+          Name:
+          <input 
+          type="text" 
+          autocomplete="off"
+          name="name">
```

Comment:
```suggestion
          <input 
            type="text" 
            autocomplete="off"
            name="name"
         >
```
Проверь код на правильное форматирование, открывающая и закрывающая дужка тега на одном уровнем атрибутов два отступа
<img width="1178" alt="formatting" src="https://user-images.githubusercontent.com/94292654/175577002-1bd760d4-03e7-4b89-b2cb-553911afedac.png">